src/domUpdates.js

```diff
@@ -137,20 +137,17 @@ const domUpdates = {
 
   //////////// HYDRATION DISPLAY SECTION /////////////////////////////////
 
-  // hydrationCardDisplay() {
-  //   // document.getElementById("hydration-user-ounces-today").innerText = `${this.currentUser.hydrationInfo.findTodaysTotalWater(this.todaysDate)}`;
-  //   // document.querySelector(".hydration-weekly-avg").innerText = `You averaged ${this.currentUser.hydrationInfo.getWeekAvgOunces()} ounces this week!`;
-  //   let dailyOz = document.querySelectorAll('.daily-oz');
-  //   let allDaysOuncesOverWeek = this.currentUser.hydrationInfo.getWeeksDailyOunces().sort((a, b) => a - b);
-  //   dailyOz.forEach((dailyOunces, i) => {
-  //     dailyOunces.innerText = allDaysOuncesOverWeek[i];
-  //   })
-  //   // input.value = "";
-  // },
-  
-  
-  // document.querySelector("#hydration-friend-ounces-today").innerText = `${this.userRepository.calculateAverageDailyWater(this.todaysDate)}`;
 
+  hydrationCardDisplay() {
+    document.getElementById("hydration-user-ounces-today").innerText = `${this.currentUser.hydrationInfo.getOuncesByDay(this.todaysDate)}`;
+    document.querySelector(".hydration-weekly-avg").innerText = `You averaged ${this.currentUser.hydrationInfo.getWeekAvgOunces()} ounces this week!`;
+    let dailyOz = document.querySelectorAll('.daily-oz');
```

Comment:
Should the variables be declared before the query selectors or do you think this doesn't matter. More of a stylistic comment than anything else.